packages/components/pages/f-checkout/src/tenants/es-ES.js

```diff
@@ -1,6 +1,318 @@
+                buttonText: 'Aceptar'
+            },
+
+            DEFAULT_CHECKOUT_ISSUE: {
+                title: 'Algo no ha ido bien',
+                message: 'Por favor, inténtelo de nuevo.',
+                buttonText: 'Aceptar'
+            },
+
+            DuplicateOrder: {
+                title: 'Este pedido no se ha realizado',
+                message: 'Es el mismo que ha hecho hace un momento y queremos comprobar que no es un error. Espere 15 minutos o cambie un artículo de su pedido',
+                buttonText: 'Cerrar',
+                buttonTextPrimary: 'Ver mis pedidos'
+            },
+
+            BasketChanged: {
+                title: 'Hemos quitado un artículo de tu pedido',
+                message: 'Puede que se le haya acabado al restaurante o que este haya actualizado su menú mientras estabas realizando el pedido, lo sentimos. Puedes volver atrás para revisar tu pedido y elegir otro artículo.',
+                buttonText: 'Volver al pedido'
+            }
+        }
+    },
+
+    warningMessages: {
+        preOrder: {
+            title: 'Por favor, tenga en cuenta que se trata de un pedido previo',
+            body: 'Por favor, compruebe el día y la hora arriba indicados'
+        }
+    },
+
+    buttonText: 'Ir al pago',
+
+    userNote: {
+        courier: {
+            delivery: {
+                title: 'Dejar una nota para el repartidor',
+                text: 'Deje una nota para el repartidor con cualquier cosa que necesite saber (por ejemplo, que el timbre no funciona).',
+                placeholder: 'por ejemplo, que el timbre no funciona'
+            },
+            collection: {
+                title: 'Dejar una nota',
+                text: 'Deje una nota para el restaurante con cualquier cosa que necesiten saber. No incluya aquí ninguna información relativa a alergias.',
+                placeholder: 'No incluya aquí detalles sobre cualquier alergia. \n\nEstamos trabajando con los restaurantes para reducir los residuos. Por favor, no pida cubiertos de plástico.'
+            },
+            dinein: {
+                title: 'Dejar una nota',
+                text: 'Deje una nota para el restaurante con cualquier cosa que necesiten saber. No incluya aquí ninguna información relativa a alergias.',
+                placeholder: 'Por ejemplo, se ruega traer los entrantes y los platos principales al mismo tiempo.'
+            }
+        },
+        order: {
+            delivery: {
+                title: 'Dejar una nota',
+                text: 'Dejar una nota for the restaurant with anything they need to know (e.g. the doorbell doesn\'t work). Do not include details about any allergies here.',
+                placeholder: 'por ejemplo, que el timbre no funciona. No incluya aquí detalles sobre cualquier alergia. \n\nEstamos trabajando con los restaurantes para reducir los residuos. Por favor, no pida cubiertos de plástico.'
```

Comment:
Nit: this grabbed my eye, should it start with a capital letter?
```suggestion
                placeholder: 'Por ejemplo, que el timbre no funciona. No incluya aquí detalles sobre cualquier alergia. \n\nEstamos trabajando con los restaurantes para reducir los residuos. Por favor, no pida cubiertos de plástico.'
```